src/JBDeadline.sol

```diff
@@ -32,9 +26,6 @@ contract JBDeadline is IJBRulesetApprovalHook {
     /// @param duration The minimum number of seconds between the time a ruleset is queued and the time it starts for it
     /// to be `Approved`.
     constructor(uint256 duration) {
-        // Ensure we don't underflow in `approvalStatusOf(...)`.
-        if (duration > block.timestamp) revert JBDeadline_DurationTooLong();
```

Comment:
Are we sure this check is no longer needed?

Reply:
its no longer needed in this contract, it seems. it was there to prevent an underflow which doesnt seem present